designs/simplified-scaffolding.md

```diff
@@ -468,7 +468,7 @@ import (
 )
 
 type MyKindReconciler struct {
-	client.Client
+	Client client.Client
```

Comment:
The design docs should not be changed
Can you please revert

Reply:
the point of this PR is to say: "the design is bad and should be changed"
if we can't agree on that then the PR makes no sense :)

... and if we can't agree on that I'd love the know why ("leave it like it is" or is there some gotcha I'm missing ?)

Reply:
This doc is a design solution implemented already.
So, the change should not be done in this specific documentation.